libraries/mbed/targets/hal/TARGET_NORDIC/TARGET_MCU_NRF51822/sleep.c

```diff
@@ -28,5 +28,5 @@ void sleep(void)
 void deepsleep(void)
 {
     sleep();
-    //   NRF_POWER->SYSTEMOFF=1;
+    NRF_POWER->SYSTEMOFF = 1;
```

Comment:
What does this change? Not needed to set it to 0 ? Why setting to 1 after calling sleep?